### PR TITLE
Fix #131 ConcurrentModificationException in LevelUp

### PR DESCRIPTION
**Pull request by EugeneButusov**

### CODE REVIEW

File: SoomlaAndroidLevelUp/src/com/soomla/levelup/LevelUp.java [hunk 1]
```diff
@@ -171,12 +171,13 @@ private static void clearCurrentState() {
 
     private static void addWorldObjectToWorlds(HashMap<String, JSONObject> worlds, JSONObject worldJSON) throws JSONException {
         String worldId = worldJSON.getString("itemId");
-        worlds.put(worldId, worldJSON);
+        HashMap<String, JSONObject> worldsInternal = new HashMap<String, JSONObject>(worlds);
```
Comment (vedi) [inline SoomlaAndroidLevelUp/src/com/soomla/levelup/LevelUp.java]: @EugeneButusov, why did you add it here?

Comment (EugeneButusov) [inline SoomlaAndroidLevelUp/src/com/soomla/levelup/LevelUp.java]: @vedi, it's a hint given by @borisspektor, crash can cause because recursive calls here.

Comment (borisspektor) [inline SoomlaAndroidLevelUp/src/com/soomla/levelup/LevelUp.java]: @EugeneButusov That's not what I meant :)
I meant that only the list at line 209(210) should be cloned because it's probably the cause of the exception.
